docs/install/create-project.md

```diff
@@ -134,49 +135,48 @@ For the `fish` shell, see [`nvm.fish`](https://github.com/jorgebucaran/nvm.fish)
 
 (install-prerequisites-yeoman-label)=
 
-#### Yeoman
+#### Yeoman and the Volto boilerplate generator
 
-Install {term}`Yeoman`.
+Install {term}`Yeoman` and the Volto boilerplate generator.
 
 ```shell
-npm install -g yo
+npm install -g yo @plone/generator-volto
 ```
 
 
 (install-prerequisites-yarn-label)=
 
-#### Yarn 3
+#### Yarn
 
-Install the latest Yarn 3 version (not the Classic 1.x one).
+Use {term}`Corepack` to enable Yarn, which was already installed with the {ref}`supported version of Node.js <install-packages-prerequisites-label>`.
 
 1.  Open a terminal and type:
 
     ```shell
     corepack enable
     ```
 
-2.  Verify that Yarn v3.x.x is installed and activated.
+In your generated Volto project from the previous step, you can find the pinned version of Yarn in its {file}`.yarnrc.yml`.
```

Comment:
@sneridagh can you verify my guess about how Yarn is pinned? This may be one of those things that seasoned JS developers "just know", but is a mystery to Python developers.

Also should we do a `yarn -v` to get the enabled version of Yarn, then tell the reader to compare that against what exists in the pin?

Reply:
I don't think that's necessary, it's pinned. Do you test if a pinned version in requirements.txt is the one that is really installed?

Reply:
Regarding the placement of the pin, it's in the `package.json` as well. `packageManager` key.

```
"packageManager": "yarn@3.2.3",
```

Reply:
> I don't think that's necessary, it's pinned. Do you test if a pinned version in requirements.txt is the one that is really installed?

That assumes everything goes right. When something goes wrong, such as the Homebrew conflict, I would compare what is pinned against what is actually used. I can remove the training wheels, and leave developers to the subsequent admonition.

Interestingly, when I ran through the installation docs:

```shell
$ yarn -v
1.22.19
$ which yarn                                                    
/Users/stevepiercy/.yarn/bin/yarn
```

Why? I have no idea. It's some remnant from 2022. So checking the system version of Yarn is misleading for a comparison anyway, as everything still works.

Fixed in cf167b78568ad21c93b787bfbb0f6f38cb10a692.